api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -167,6 +167,10 @@ const (
 	// of the cluster will be removed, including image registry storage, ingress dns records, load balancers, and persistent storage.
 	CleanupCloudResourcesAnnotation = "hypershift.openshift.io/cleanup-cloud-resources"
 
+	// CleanupORCImageResourcesAnnotation is an annotation that indicates whether an OpenStack Image should be
```

Comment:
Could you add the why here?

Reply:
Done. Next patchset will have a longer description explaining why.